core/plugin_manager.js

```diff
@@ -21,7 +21,7 @@ goog.provide('Blockly.PluginManager');
 Blockly.PluginManager = function() {
   /**
    * A map of the plugins registered with the workspace, mapped to id.
-   * @type {!Object<string, !Blockly.PluginManager.PluginData>}
+   * @type {!Object<string, !Blockly.PluginManager.PluginDatum>}
    */
   this.pluginData_ = {};
```

Comment:
Since we changed the type, should we change this as well?

Reply:
I changed the type from `PluginData` to `PluginDatum` because datum is the singular of data. My intention was to use datum for a single plugin object bag and data to refer to a collection of them.